model/schema/variation.yaml

```diff
@@ -101,9 +101,7 @@ classes:
       - reference_sequence
       - variant_sequence
       - consequence
-    slot_usage:
-      evidence_code:
-        required: false
+      - sequence_of_reference_accession_number
```

Comment:
We have `assembly` and `chromosome` slots on `VariantGenomeLocation` - is this required, and should it really be on the base class?